README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
+   + here is the link to watch the best Trek while stuck inside. [You'll thank me](https://www.netflix.com/title/70158330?source=35) 
+* **Romantic comedies**
+    + Before you ask, it's *You've Got Mail* and I do not like *Sleepless in Seattle*
+---------
+I love travelling--before coming to the MAPSS program, I travelled for four months--one of my favorite places was Georgia--but I didn't take any photos of me there so here's one from Armenia.   
```

Comment:
I'm not sure Markdown supports extended ellipses. One of the drawbacks to Markdown is it is relatively simplistic about how you can style documents